includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1055,6 +1055,67 @@ public function generate_create_intent_request( $order, $prepared_source ) {
 		return apply_filters( 'wc_stripe_generate_create_intent_request', $request, $order, $prepared_source );
 	}
 
+	/**
+	 * Create the level 3 data array to send to Stripe when making a purchase.
+	 *
+	 * @param WC_Order $order The order that is being paid for.
+	 * @return array          The level 3 data to send to Stripe.
+	 */
+	public function get_level3_data_from_order( $order ) {
+		// Return an empty array if there's no store postcode (since it's
+		// required for level3 data), or if the version is less than 3.0
+		// (because it's not possible to get a store postcode before WC 3.0).
+		if ( WC_Stripe_Helper::is_wc_lt( '3.0' ) ) {
+			return array();
+		}
```

Comment:
>Return an empty array if there's no store postcode

This seems to not be true anymore?

Reply:
Updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1130/commits/e38959829e18a9364271ef2e477582c1c9ea0c0d